File: src/Maruko/Application/Servers/Dto/PagedResultDto.cs
namespace Maruko.Core.Application.Servers.Dto
{
    /// <summary>
    /// 分页返回结果
    /// </summary>
    public class PagedResultDto
    {
        /// <summary>
        /// 总条数
        /// </summary>
        public long TotalCount { get; set; }

        /// <summary>
        /// 数据
        /// </summary>
        public object Datas { get; set; }

        public  PagedResultDto(long total, object data)
        {
            TotalCount = total;
            Datas = data;
        }
    }
}

File: act/act-framework/act-framework-core/src/main/java/com/act/core/application/PagedResultDto.java
package com.act.core.application;

import lombok.Data;

/*
分页查询返回对象
 */
@Data
public class PagedResultDto {
    private long totalCount;
    private Object datas;

    public PagedResultDto(long total, Object data) {
        totalCount = total;
        datas = data;
    }
}
